### PR TITLE
🐕 Fixing issues

### LINKED ISSUE #482: Re-add pagination into Admin Appointments Page
This was a previous [PR](https://github.com/hack4impact-uiuc/mentee/pull/373) and now we just need to fix this to have a more robust chronological implementation. Check out this [website ](https://www.codementor.io/@arpitbhayani/fast-and-efficient-pagination-in-mongodb-9095flbqr)to see how we can improve it as the current way it shows appointments from oldest to newest instead of newest to oldest.

### PR DESCRIPTION
**Pull request by SolMon9099**
## Description :star2:

Fixes #482, #765, #847

### CODE REVIEW

**Review by robert-dm — APPROVED** [review]

**Review by robert-dm — APPROVED** [review]

File: frontend/src/components/pages/AdminAppointmentData.js [hunk 1]
```diff
@@ -138,6 +253,103 @@ function AdminAppointmentData() {
...
+                  <svg
+                    viewBox="64 64 896 896"
+                    focusable="false"
+                    data-icon="left"
+                    width="1em"
+                    height="1em"
+                    fill="currentColor"
+                    aria-hidden="true"
+                  >
+                    <path d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 000 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"></path>
```
Comment (robert-dm) [inline frontend/src/components/pages/AdminAppointmentData.js:291]: @SolMon9099 pls check this.

File: frontend/src/components/pages/AdminAppointmentData.js [hunk 2]
```diff
@@ -138,6 +253,103 @@ function AdminAppointmentData() {
...
+                  <svg
+                    viewBox="64 64 896 896"
+                    focusable="false"
+                    data-icon="right"
+                    width="1em"
+                    height="1em"
+                    fill="currentColor"
+                    aria-hidden="true"
+                  >
+                    <path d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"></path>
```
Comment (robert-dm) [inline frontend/src/components/pages/AdminAppointmentData.js:345]: @SolMon9099 pls check what is it

**Review by robert-dm — APPROVED** [review]

File: frontend/src/components/pages/AdminAppointmentData.js (hunk 1 above)
Comment (SolMon9099) [reply frontend/src/components/pages/AdminAppointmentData.js:291]: This is svg icon

File: frontend/src/components/pages/AdminAppointmentData.js (hunk 2 above)
Comment (SolMon9099) [reply frontend/src/components/pages/AdminAppointmentData.js:345]: This is svg icon

**Pull request merged**